src/qs_linres_module.F

```diff
@@ -240,7 +240,10 @@ SUBROUTINE linres_calculation_low(qs_env)
       lr_section => section_vals_get_subs_vals(qs_env%input, "PROPERTIES%LINRES")
       CALL section_vals_get(lr_section, explicit=lr_calculation)
 
-      IF (lr_calculation) THEN
+      ALLOCATE (linres_control)
```

Comment:
The `linres_control` is not getting deallocated, which leads to a [memory leak](https://storage.googleapis.com/cp2k-ci/run-cp2k-sdbg-cda7b2e3_report.txt).

Reply:
Thanks. Fixed.